components/lookup/lookup.jsx

```diff
@@ -87,7 +85,7 @@ const Lookup = createReactClass({
 		 */
 		filterWith: PropTypes.func,
 		/**
-		 * If true, the menu is constrained to the window and may be flipped up. Has no effect if `isInline` is `true`.
+		 * If true, the menu is constrained to the window and may be flipped up. Has no effect if `isInline` is `true`. In other components, its opposite is used `hasStaticAlignment`.
```

Comment:
Do we need to change the description to include `menuPosition="relative" vs. isInline`?

Reply:
Lookup still uses the old multiple sub-render `isInline` prop. I'm hesitant to change anything on a deprecated component. What do you think?